src/utilcode/util.cpp

```diff
@@ -1288,7 +1288,15 @@ int GetCurrentProcessCpuCount()
     // the case where there are more than 64 processors, so we will return a
```

Comment:
We are saying the same thing - without enabling CPU groups you always get 64 CPUs max right now, on Windows or not.

It sounds like you said you already implemented CPU groups on Linux even though Linux probably doesn't have the concept of CPU groups. So if we enabled your code, it would work out the same way on Windows and on Linux. Thus no need for this PR. Is that correct?